src/types/SdkTypes.res

```diff
@@ -302,6 +302,9 @@ type hyperParams = {
   defaultView: bool,
   launchTime?: float,
   sdkVersion: string,
+  device_model: option<string>,
+  os_type: option<string>,
+  os_version: option<string>,
```

Comment:
can you make the type string instead, anyways from native side you are always collecting these and putting them in hyper params
```suggestion
  device_model: string,
  os_type: string,
  os_version: string,
```

Reply:
any way both serve the same purpose, right?